be/src/main/java/com/savethepet/model/entity/Role.java

```diff
@@ -0,0 +1,17 @@
+package com.savethepet.model.entity;
+
+import org.springframework.security.core.GrantedAuthority;
+
+/**
+ * @author Alexey Klimov
+ * all roles that exists in application
+ */
+public enum Role implements GrantedAuthority {
+    USER,
+    ADMIN;
```

Comment:
This role is not being used yet, so maybe we can delete this unused variable at this stage